models/members.js

```diff
@@ -170,11 +173,44 @@ const addArchiveRoleToMembers = async (userId) => {
   }
 };
 
+/**
+ * Adds default archived role
+ * @return {Promise<usersMigrated|Object>}
+ */
+const addDefaultArchivedRole = async () => {
+  try {
+    const userSnapShot = await userModel.get();
+    const migratedUsers = [];
+
+    const usersArr = [];
+
+    userSnapShot.forEach((doc) => usersArr.push({ id: doc.id, ...doc.data() }));
```

Comment:
Does this work for 1 million users?

Reply:
I think we should paginate the queries to firestore